src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {
+    Order add(Order order);
+
+    Optional<List<Order>> getByUser(User user);
```

Comment:
```suggestion
   List<Order> getByUser(User user);
```
dao would return empty list if user doesn't have orders